client/app/bundles/course/assessment/submission/pages/SubmissionEditIndex/index.jsx

```diff
@@ -500,6 +521,7 @@ function mapStateToProps({ assessments: { submission } }) {
     topics: submission.topics,
     isLoading: submission.submissionFlags.isLoading,
     isSaving: submission.submissionFlags.isSaving,
+    savingStatus: submission.submissionFlags.savingStatus,
```

Comment:
I think we are passing too many props here and these props are not needed in SubmissionEditForm and SubmissionEditStepForm. Why not just get the status "of the specific answer" directly in the SubmissionAnswer component?  This way, you dont need to pass a huge savingStatus object over and over and instead get the specific savingStatus for the answer right away. Let me know if you dont understand.

Reply:
done